validator/testdata/feature_tests/ads.html

```diff
@@ -116,6 +116,7 @@
         <option>csa</option>
         <option>custom</option>
         <option>eas</option>
+        <option>directadvert</option>
```

Comment:
I don't think you need to change this file. It's used for validator test only. 

to @choumx I see you removed the validator test to our example/* files in #10292, do we ever update files in `validator/testdata/feature_tests` folder?